.github/workflows/ci.yaml

```
@@ -0,0 +1,44 @@
+---
+
+name: Continuous integration
+
+on:
+  push:
+
+jobs:
+  main:
+    name: Continuous integration
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+      TX_USR: ${{ secrets.TX_USR }}
+      TX_PWD: ${{ secrets.TX_PWD }}
+      TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+      TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+    steps:
+      - name: Install packages
+        run: sudo apt-get install -y gettext python3-virtualenv
+      - uses: actions/checkout@v1
+      - name: Build
+        run: make build
+      - name: Lint
+        run: make check
+      - name: Tests
+        run: make test
+      - name: Docker compose logs
+        if: ${{ always() }}
```

`{{` are not needed ...
```suggestion
        if: always()
```